examples/src/memory/entity_memory_inspection.ts

```diff
@@ -0,0 +1,43 @@
+import { OpenAI } from "langchain/llms/openai";
+import {
+  EntityMemory,
+  ENTITY_MEMORY_CONVERSATION_TEMPLATE,
+} from "langchain/memory";
+import { LLMChain } from "langchain/chains";
+
+const memory = new EntityMemory({
+  llm: new OpenAI({ temperature: 0 }),
+});
+const model = new OpenAI({ temperature: 0.9 });
+const chain = new LLMChain({
+  llm: model,
+  prompt: ENTITY_MEMORY_CONVERSATION_TEMPLATE,
```

Comment:
Should be its own chain in the future or an option on `ConversationChain` @nfcampos?